### PR TITLE
update playground doc

### PR DESCRIPTION
**Pull request by kaaaaaaang**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by KanShiori — APPROVED** [review]

File: tiup/tiup-faq.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ TiUP 暂时不支持外部开发的组件，但是 TiUP Team 已经制定了 TiU
 
 ## tiup-playground 和 tiup-cluster 有什么区别？
 
-TiUP Playground 组件主要定位是快速上手和搭建单机的开发环境，支持 Linux/macOS，要运行一个指定版本的 TiUP 集群更加简单。TiUP Cluster 组件主要是部署生产环境集群，通常是一个大规模的集群，还包含运维相关操作。
+TiUP Playground 组件主要定位是快速上手和搭建单机的开发环境，支持 Linux/macOS，要运行一个指定版本的 TiUP 集群更加简单。TiUP Cluster 组件主要是部署生产环境集群，通常是一个大规模的集群，还包含运维相关操作。TiUP Playground 部属的 TiDB 集群会缺失部分功能和运维操作，并不建议用于完整的功能性测试和稳定性测试。
```
Comment (hfxsd) [inline tiup/tiup-faq.md:23]:
```suggestion
TiUP Playground 组件主要定位是快速上手和搭建单机的开发环境，支持 Linux/macOS，要运行一个指定版本的 TiUP 集群更加简单。TiUP Cluster 组件主要是部署生产环境集群，通常是一个大规模的集群，还包含运维相关操作。TiUP Playground 部属的 TiDB 集群可能会缺失部分功能和运维操作，不建议用于完整的功能性测试和稳定性测试。
```

**Review by hfxsd — APPROVED** [review]

File: tiup/tiup-faq.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ TiUP 暂时不支持外部开发的组件，但是 TiUP Team 已经制定了 TiU
 
 ## tiup-playground 和 tiup-cluster 有什么区别？
 
-TiUP Playground 组件主要定位是快速上手和搭建单机的开发环境，支持 Linux/macOS，要运行一个指定版本的 TiUP 集群更加简单。TiUP Cluster 组件主要是部署生产环境集群，通常是一个大规模的集群，还包含运维相关操作。
+TiUP Playground 组件主要定位是快速上手和搭建单机的开发环境，支持 Linux/macOS，要运行一个指定版本的 TiUP 集群更加简单。TiUP Cluster 组件主要是部署生产环境集群，通常是一个大规模的集群，还包含运维相关操作。TiUP Playground 部属的 TiDB 集群可能会缺失部分功能和运维操作，不建议用于完整的功能性测试和稳定性测试。
```
Comment (qiancai) [inline tiup/tiup-faq.md:23]:
```suggestion
TiUP Playground 组件主要定位是快速上手和搭建单机的开发环境，支持 Linux/macOS，要运行一个指定版本的 TiUP 集群更加简单。TiUP Cluster 组件主要是部署生产环境集群，通常是一个大规模的集群，还包含运维相关操作。TiUP Playground 部署的 TiDB 集群可能会缺失部分功能和运维操作，不建议用于完整的功能性测试和稳定性测试。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**